Foundation/CharacterSet.swift

```diff
@@ -486,7 +486,10 @@ extension CharacterSet : _ObjectTypeBridgeable {
     
     @_semantics("convertToObjectiveC")
     public func _bridgeToObjectiveC() -> NSCharacterSet {
-        return _wrapped
+        switch _wrapped.__wrapped {
```

Comment:
@phausler I'd like to run this against you and ask for advice.

I'm not sure if this solutionis ok.

Perhaps it makes more sense to keep `return _wrapped` here, and add
```swift
override var _cfObject: CFType {
    return _mapUnmanaged { unsafeBitCast($0, to: CFType.self) }
}
```
to `_SwiftNSCharacterSet`

The problem I'm trying to solve is that we return `_wrapped` here, which is `_SwiftNSCharacterSet`.
Somewhere later, we access `._cfObject` in this object, which does `unsafeBitCast(self, to: CFType.self)`.
But this doesn't work correctly, because `_SwiftNSCharacterSet` has different structure than `CFCharacterSet`.